web/static/js/components/thesis-editor.js

```diff
@@ -223,6 +264,44 @@ class ThesisEditor extends React.Component {
     }
   }
 
+  importContent (t, d, page) {
```

Comment:
@jamonholmgren , again, copied this a different function. I agree though, your argument variable naming sucks.